tests/test_unit/test_tracking_utils.py

```diff
@@ -17,26 +17,23 @@
 
 
 @pytest.mark.parametrize(
-    "prev_frame_id, current_frame_id, expected_output",
+    "prev_frame_id, current_frame_id, n_gt, expected_output",
```

Comment:
The test of an "appearing crab" fails if the number of ID switches is expected to be 0


This is the test I tried:
```
@pytest.mark.parametrize(
    "prev_frame_id, current_frame_id, n_gt, expected_output",
    [
        ([[4, 3, 2, 1]], [[5, 4, 3, 2, 1]], 5, 0),
    ],
)
def test_count_identity_switches(
    prev_frame_id, current_frame_id, n_gt, expected_output
):
    assert (
        count_identity_switches(prev_frame_id, current_frame_id, n_gt)
        == expected_output
    )
```